atom/browser/api/atom_api_web_contents.cc

```diff
@@ -404,6 +404,11 @@ void WebContents::OnCreateWindow(const GURL& target_url,
 content::WebContents* WebContents::OpenURLFromTab(
     content::WebContents* source,
     const content::OpenURLParams& params) {
+  if (params.disposition == SAVE_TO_DISK) {
```

Comment:
This changes current behavior and not all users want to have the ability download things from the app, we can probably just make it explicit in the `new-window` event and document it.